styles/index.scss

```diff
@@ -15,13 +15,28 @@ $heading-font: 'Nunito', 'Avenir', sans-serif;
   width: 100%;
 }
 
+.top-photo {
+  padding: 0px 0px 0px 15px;
+  position: absolute;
+  left: 47%;
+  overflow: hidden;
+  width: 45%;
+}
+
 .section{
-  width: 100%;
+  position: relative;
+  min-width: 300px;
+}
+
+#home-details{
+  width: 45%;
```

Comment:
same here. instead of specifying a width, i would use the classes from the other pages to make use of flexbox